CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Changelog
 
+# 0.8.2 (2019-04-11)
+
+## Migration Disclaimer
+
+If you have the following set in your M3Query or M3Coordinator config, you should remove it. This is due to a bug in the LRU conversion cache within M3Query that is fixed in this release (see below).
+```yaml
+cache:
+  queryConversion:
+    size: <int>
+```
+
+## New Features
+
+- **M3DB**: Add Jaeger tracing to M3DB QueryIDs path (#1506)
+
+## Performance
+
+- **M3DB**: Add postingsList caching for FieldQuery (#1530)
+- **M3DB**: Add write & read latency histogram metrics to M3DB client and emit from M3Coordinator (#1533)
```

Comment:
didnt we add a coordinator grafana dashboard too or am I thinking of something else

Reply:
Haven't landed that yet. Will land it this week once i get some time.